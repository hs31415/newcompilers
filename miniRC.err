Line: 8 Invalid token.