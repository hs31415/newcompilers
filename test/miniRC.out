<id, miniRC>
 <=>
 <function>
 <(>
 <integer>
<id, delta>
 <)>
 <{>
 <integer>
<id, a>
 <[>
<INT, 5>
 <]>
 <[>
<INT, 10>
 <]>
 <,>
<id, b>
 <[>
<INT, 5>
 <]>
 <[>
<INT, 10>
 <]>
 <,>
<id, i>
 <,>
<id, j>
 <;>
 <for>
 <(>
<id, i>
 <=>
<NUM, 0>
 <;>
<id, i>
 <<>
<INT, 5>
 <;>
<id, i>
 <+>
 <+>
 <)>
 <for>
 <(>
<id, j>
 <=>
<NUM, 0>
 <;>
<id, j>
 <<>
<INT, 10>
 <;>
<id, j>
 <+>
 <+>
 <)>
 <{>
Line: 4 Invalid token.Line: 4 Invalid token.<id, a>
 <[>
<id, i>
 <]>
 <[>
<id, j>
 <]>
 <=>
<id, i>
 <+>
<id, j>
 <;>
 <if>
 <(>
<id, i>
 <>>
<id, j>
 <)>
<id, b>
 <[>
<id, i>
 <]>
 <[>
<id, j>
 <]>
 <=>
<id, i>
 <*>
<id, delta>
 <+>
<id, j>
 <;>
 <else>
<id, b>
 <[>
<id, i>
 <]>
 <[>
<id, j>
 <]>
 <=>
<id, i>
 <->
<id, j>
 </>
<id, delta>
 <;>
 <}>
 <integer>
<id, sum>
 <;>
<id, sum>
 <=>
<id, i>
 <=>
<id, j>
 <=>
<NUM, 0>
 <;>
 <repeat>
 <{>
 <if>
 <(>
<id, i>
 <>>
<id, j>
 <)>
 <{>
<id, j>
 <+>
 <+>
 <;>
<id, sum>
 <->
 <->
 <;>
 <}>
 <else>
<id, i>
 <+>
 <+>
 <;>
 <if>
 <(>
<id, i>
 <<>
<INT, 5>
 <&&>
<id, j>
 <<>
<INT, 10>
 <)>
<id, sum>
 <=>
<id, sum>
 <+>
<id, a>
 <[>
<id, i>
 <]>
 <[>
<id, j>
 <]>
 <*>
<id, b>
 <[>
<id, i>
 <]>
 <[>
<id, j>
 <]>
 <;>
 <else>
<id, break>
 <;>
 <}>
 <until>
 <(>
<id, sum>
 <>=>
<INT, 100>
 <)>
 <;>
 <return>
 <(>
<id, sum>
 <)>
 <;>
 <}>
